vimrc

```diff
@@ -29,6 +29,16 @@ endif
 
 filetype plugin indent on
 
+" Softtabs, 2 spaces
+set tabstop=2
+set shiftwidth=2
+set shiftround
+set expandtab
+set softtabstop=2
```

Comment:
That's true, I suggest placing them under ".vim/ftplugin/go.vim" and use "setlocal" instead of "set" to have them locally done per go files

---
Teo Ljungberg

> On 20 nov. 2015, at 19:01, Geoff Harcourt <notifications@github.com> wrote:
> 
> In vimrc:
> 
> > @@ -29,6 +29,16 @@ endif
> >  
> >  filetype plugin indent on
> >  
> > +" Softtabs, 2 spaces
> > +set tabstop=2
> > +set shiftwidth=2
> > +set shiftround
> > +set expandtab
> > +set softtabstop=2
> If these were an autocmd for Go with setlocals, all these settings would be Go-only, correct?
> 
> —
> Reply to this email directly or view it on GitHub.
>